app/src/chat_api.py

```diff
@@ -226,8 +218,8 @@ async def query(request: QueryRequest) -> QueryResponse:
                 detail=f"Chat history for existing session not found: {request.session_id}",
             )
 
-        _save_message(
-            ChatMessage(request.session_id, "user", request.message, request_msg.timestamp)
+        db_session.add(
+            ChatMessage(session_id=request.session_id, role="user", content=request.message)
```

Comment:
non-blocking: should we have an enum for the roles "user" and "assistant" defined somewhere?

Reply:
I considered that, but since we're prototyping/piloting, I left it open in case we want to experiment with other roles.